docs/administration/recent_activity/recent_activity.md

```diff
@@ -0,0 +1,231 @@
+A context group counts as one item in regard to `activity_logs_limit` configuration and `ActivityLogService::findGroups`'s `$limit` argument.
+
+To open a context group, use `ActivityLogService::prepareContext` which has 2 arguments:
+
+* `$source` is describing (usually through a short identifier) what is triggering the set of actions. For example, some already existing source are `web` (like for actions from the Back Office), `graphql`, `rest` and `migration`.
+* `$description` is an optional more specific contextualisation. For example, `migration` context source is associated with the migration file name in its context description.
+
+To close a context group, use `ActivityLogService::dismissContext`.
+
+In the following example, several actions are logged into one context group, even actions triggered by cascade outside the piece of code:
+
+- `my_feature`
+    - `init`
+    - `create`
+    - `publish`
+    - `simulate`
+    - `complete`
+
+``` php
+[[= include_file('code_samples/recent_activity/src/Command/ActivityLogContextTestCommand.php', 63, 83) =]]
+```
+
+Context groups can't be nested.
+If a new context is prepared while a context is already grouping log entries, this new context will be ignored. (Nested context might be supported in the future.)
+To start a new context, make sure to previously dismiss the existing one.
+
+As the context are not really user-friendly, by default, contexts are not shown in the Back Office.
+A context group is folded below its first entry.
+
+#### List
+
+To display your log entry, if your object's PHP class isn't already covered, you'll have to:
+
+* implement `ClassNameMapperInterface` to associate the class name with an identifier,
+* eventually create a `PostActivityListLoadEvent` subscriber if you need to load the object for the template,
+* create a template to display this class log entries.
+
+You can have a template:
+
+* specific to a class identifier and placed in `templates/themes/<theme>/activity_log/ui/<identifier>.html.twig`
+* specific to an action on an identifier and placed in `templates/themes/<theme>/activity_log/ui/<identifier>/<action>.html.twig`
+
+Template existence is tested in reverse order: If there is no action specific template, the identifier's default is used.
+For the same identifier, you could have specific templates for few actions, and a default one for the remaining actions.
+
+A default template is used if no template is found for the identifier.
+The built-in default template `@ibexadesign/activity_log/ui/default.html.twig` has an empty `activity_log_description_widget` block and display nothing for unknown objects.
+Your template can extend `@ibexadesign/activity_log/ui/default.html.twig`, and only redefine the `activity_log_description_widget` block for your objects.
+
+First, follows an example of a default template overriding the one from the bundle. It can be used at development time as a fallback for classes not yet mapped.
```

Comment:
```suggestion
First, follow an example of a default template overriding the one from the bundle. It can be used at development time as a fallback for classes not yet mapped.
```